hc/settings.py

```diff
@@ -14,7 +14,7 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 HOST = "localhost"
-SECRET_KEY = os.environ.get("HC_SECRET_KEY")
+SECRET_KEY = '---'
```

Comment:
why are you putting a hard coded secret key

Reply:
maintain the code convention

Reply:
Oh, forgot to change it. I will fix it.

Reply:
I have fixed it.